### ISSUE TITLE
SSL/TLS endpoint support request

### ISSUE THREAD

**Issue by nsteinmetz** [senx/warp10-plugin-mqtt#2]
Hello,

Right now, plugin support only this method:

```java
    //
    // Create MQTT client
    //
    MQTT mqtt = new MQTT();

    mqtt.setCleanSession(this.mqttcleansession);
    mqtt.setHost(this.mqtthost, this.mqttport);
```

but anoter exists:

```
MQTT mqtt = new MQTT();
mqtt.setHost("localhost", 1883);
// or 
mqtt.setHost("tcp://localhost:1883");
```

So I naively changed for:

```java

    //
    // Create MQTT client
    //
    MQTT mqtt = new MQTT();

    mqtt.setCleanSession(this.mqttcleansession);

    if (this.mqtthost.startsWith("tcp://") || this.mqtthost.startsWith("ssl://") || this.mqtthost.startsWith("sslv3://") || this.mqtthost.startsWith("tls://")  || this.mqtthost.startsWith("tlsv1.1://") || this.mqtthost.startsWith("tlsv1.2://")  || this.mqtthost.startsWith("tlsv1.3://")) {
      mqtt.setHost(String.format("%s:%s", this.mqtthost, this.mqttport));
    } else {
      mqtt.setHost(this.mqtthost, this.mqttport);
    }
    System.out.println(mqtt.getHost());
```

For non ssl/tls connection, I got in the logs:
```
tcp://host:1883
```
But nothing when I use ssl/tls

It seems it requires some SSLContext and that's where we hit my Java limits 😉 

from https://github.com/fusesource/mqtt-client#using-ssl-connections:

> ### Using SSL connections
>
> If you want to connect over SSL/TLS instead of TCP, use an "ssl://" or
> "tls://" URI prefix instead of "tcp://" for the `host` field. For finer
> grained control of which algorithm is used. Supported protocol values are: 
>
> * `ssl://`    - Use the JVM default version of the SSL algorithm.
>* `sslv*://`  - Use a specific SSL version where `*` is a version
>  supported by your JVM.  Example: `sslv3`
>* `tls://`    - Use the JVM default version of the TLS algorithm.
>* `tlsv*://`  - Use a specific TLS version where `*` is a version
> supported by your JVM.  Example: `tlsv1.1`
>
>The client will use the default JVM `SSLContext` which is configured via JVM
>system properties unless you configure the MQTT instance using the
>`setSslContext` method.
>
>SSL connections perform blocking operations against internal thread pool
>unless you call the `setBlockingExecutor` method to configure that executor
>they will use instead.

**Comment by nsteinmetz** [comment]
@pi-r-p told me to look at https://github.com/senx/warp10-platform/blob/master/warp10/src/main/java/io/warp10/SSLUtils.java but I don't get know what to do with it 😢

**Comment by nsteinmetz** [comment]
Fixed with cf4ee08122109081fbc99365d9815f708001e301
